proto/metadata.proto

```diff
@@ -20,3 +20,7 @@ message CompareInMetadata {
     // consts represents all non-null const args in repeated Datum format.
     optional bytes consts = 2;
 }
+
+message KeepNullMetadata {
```

Comment:
prefer to name it for functions, i.e. IsTrueMetadata

Reply:
This name is to be consistent with a similar implementation [before](https://github.com/pingcap/tipb/pull/161/files). We use the parameters in the expression variable to name instead of the name of the expression. Besides, `keepNull` can be used in more expression, such as `isTrue` , `isFalse`. What's your opinion? @breeswish

Reply:
InUnionMetadata was named as `InUnion` because not all cast functions uses it so that it is not suitable to name it `CastMetadata`.

If multiple (but not many) functions shares it, prefer to duplicate it, like `IsTrueMetadata` and `IsFalseMetadata`. The reason of the duplication is that each SIG can only associate with one metadata, so that when more properties are needed in future, it can be added easily.

Reply:
Got it. @breeswish PTAL